octoprint_findmymrbeam/__init__.py

```diff
@@ -186,7 +186,9 @@ def _get_local_ping_data(self):
 
 	def get_settings_defaults(self):
 		return dict(enabled=True,
-					url="https://find.mr-beam.org/registry",
+					# url="https://find.mr-beam.org/registry",
+					# Revert this change before beta release
```

Comment:
please create a ticket about it and mention it in the comment

Reply:
addressed